src/acl.c

```diff
@@ -2086,8 +2070,8 @@ sds ACLLoadFromFile(const char *filename) {
         argv = sdssplitlen(lines[i],sdslen(lines[i])," ",1,&argc);
         if (argv == NULL) {
             errors = sdscatprintf(errors,
-                     "%s:%d: unbalanced quotes in acl line. ",
-                     server.acl_filename, linenum);
+                                  "%s:%d: unbalanced quotes in acl line. ",
+                                  server.acl_filename, linenum);
```

Comment:
If we can get away with it, let's not touch these lines to preserve git history.

Reply:
ah, this was probably just an editor induced change that I missed when cleaning it up.  will revert.

Reply:
fixed